src/repositories/tags.repository.ts

```diff
@@ -23,3 +34,14 @@ export const update = (tagId: string, newTagData: Partial<Tag>) => {
 export const deleteById = (tagId: string) => {
   return TagModel.findByIdAndDelete(tagId);
 };
+
+export const getByName = (name: string) => {
+  return TagModel.findOne({ name: name });
+};
+
+export const incrementStatsByName = (name: string) => {
```

Comment:
add function decrement 
When we created tweet with tag, but after that modificated it and removed some tags from tweet